### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by varnita619**
- added login ,
- signup ,
- logout authentication

### CODE REVIEW

**Review by pritam-kr — APPROVED** [review]
Celan UI

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=VarnitaMakrariya
```
Comment (manish591) [inline .env:1]: Don't commit `env`. Add it in the gitignore

File: src/Services/authServices.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                });
+                navigate("/signup")
+
+                //Updating initial States
+                dispatch(authInfo(foundUser));
+                dispatch(getToken(encodedToken));
+            }
+
+        } catch (error) {
+            toast.error("Error occured in logging in!", { position: "top-right" });
```
Comment (manish591) [inline src/Services/authServices.js:35]: You can define toast position in the Toaster component itself so that you don't need to write it elsewhere

**Review by manish591 — APPROVED** [review]
The code is looking good.

File: .env (hunk 1 above)
Comment (varnita619) [reply .env:1]: Done

**Pull request merged**